mysql/infoschema.go

```diff
@@ -172,25 +172,22 @@ type schemaAndName struct {
 	name   string
 }
 
-func getTables(db *sql.DB) ([]schemaAndName, error) {
-	ignored := make(map[string]bool)
-	// Ignore all system tables: we just want to convert user tables.
-	for _, s := range []string{"information_schema", "mysql", "performance_schema", "sys"} {
-		ignored[s] = true
-	}
-	q := "SELECT table_schema, table_name FROM information_schema.tables where table_type = 'BASE TABLE'"
-	rows, err := db.Query(q)
+// getTables return list of tables in the selected database.
+// Note that sql.DB already effectively has the dbName
+// embedded within it (dbName is part of the DSN passed to sql.Open),
+// but unfortunately there is no way to extract it from sql.DB.
+func getTables(db *sql.DB, dbName string) ([]schemaAndName, error) {
+	q := "SELECT table_name FROM information_schema.tables where table_type = 'BASE TABLE' and table_schema=?"
+	rows, err := db.Query(q, dbName)
```

Comment:
I think `// In MySQL, schema is the same as database name.` needs to be put in front of this line instead of the previous place.

Reply:
Got it. Done.